modules/pytket-qiskit/pytket/extensions/qiskit/qiskit_convert.py

```diff
@@ -427,6 +440,11 @@ def append_tk_command_to_qiskit(
         qcirc.append(new_gate, qargs)
         return qcirc
 
+    if optype == OpType.CU3:
+        params = [param_to_qiskit(p, symb_map) for p in op.params] + [0]
+        params.append(0)
```

Comment:
should 0 be appended twice?

Reply:
Ooh, not sure how that worked...

Reply:
My first thought was that it wasn't being tested, but it is, so it must be that qiskit just ignores the extra parameter.